### DISCUSSION TITLE
Blitz and Quirrel Debugging

### DISCUSSION THREAD

**Discussion by taylorpreston** [quirrel-dev/quirrel#791] [Q&A]
Hey. Is this the correct way to debug Blitz and Quirrel?

I am trying to configure the launch.json in my project. I have the bellow configured and am really just looking for any advise on debugging my application. First time using Quirrel and am stoked with how easy it is
```
{
  "version": "0.2.0",
  "configurations": [
    {
      "type": "node",
      "request": "launch",
      "name": "Blitz node debug",
      "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/concurrently",
      "runtimeArgs": ["--raw", "blitz dev", "./node_modules/.bin/quirrel"],
      "port": 9229,
      "console": "integratedTerminal"
    }
  ]
}
{
  "version": "0.2.0",
  "configurations": [],
  "compounds": []
}
```

**Comment by Skn0tt** [comment]
Hi Taylor! I tried your `launch.json`, removed the second JSON at the bottom of your script, omitted the `port` field (VSCode told me that doesn't exist 🤷) and it works fine for me:

```
{
  "version": "0.2.0",
  "configurations": [
    {
      "type": "node",
      "request": "launch",
      "name": "Blitz node debug",
      "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/concurrently",
      "runtimeArgs": ["--raw", "blitz dev", "./node_modules/.bin/quirrel"],
      "console": "integratedTerminal"
    }
  ]
}
```

Can you verify this doesn't work for you? Are you using `npm` or some other package manager?
